src/main/java/com/atlassian/shele/shelePlugin/ao/IssueService.java

```diff
@@ -0,0 +1,81 @@
+import com.atlassian.jira.event.type.EventType;
+import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
+import lombok.SneakyThrows;
+import net.java.ao.Query;
+import org.ofbiz.core.entity.GenericValue;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Component
+public class IssueService {
+
+    private final ActiveObjects activeObjects;
+    private final MapIssueMapper mapper;
+
+    @Autowired
+    public IssueService(@ComponentImport ActiveObjects activeObjects, MapIssueMapper mapper) {
+        this.activeObjects = activeObjects;
+        this.mapper = mapper;
+    }
+
+    @SneakyThrows
+    public void persistIssueEntity(IssueEvent event) {
+        IssueEntity issueEntity = activeObjects.create(IssueEntity.class);
+        issueEntity.setAuthorIssue(event.getIssue().getCreator().getName());
+        issueEntity.setCreatedTime(event.getTime());
+        issueEntity.setIssueId(Math.toIntExact(event.getIssue().getId()));
+        issueEntity.setProjectId(Math.toIntExact(event.getProject().getId()));
+        if (event.getEventTypeId().equals(EventType.ISSUE_COMMENT_EDITED_ID)){
+            issueEntity.setNewField(event.getComment().getBody());
+            issueEntity.setField("Comment");
+        }else if (event.getEventTypeId().equals(EventType.ISSUE_COMMENTED_ID)){
+            issueEntity.setNewField(event.getComment().getBody());
+        }else {
+            List<GenericValue> list = event.getChangeLog().getRelated("ChildChangeItem");
+            for (GenericValue value : list) {
+                issueEntity.setField(String.valueOf(value.get("field")));
+                issueEntity.setNewField(String.valueOf(value.get("newstring")));
+                issueEntity.setPrevField(String.valueOf(value.get("oldstring")));
+            }
+
+            issueEntity.setWorkFlow(String.valueOf(ComponentAccessor.getWorkflowManager().getWorkflow(event.getIssue()).getDisplayName()));
+            issueEntity.setEvent(event.getEventTypeId());
+            issueEntity.save();
+        }
+        issueEntity.save();
+    }
+
+
+    public Optional<String> stringOptional(IssueEvent eventIssue) {
+        Optional<String> stringOptional = Arrays.stream(activeObjects.find(ProjectEntity.class, Query.select().where("PROJECT=?", eventIssue.getProject().getId()))).map(ProjectEntity::getEventTypeId)
+                .findFirst();
+        if (stringOptional.isPresent()) {
+            return stringOptional;
+        }
+        return null;
+
+    }
+
+    public List<IssueDTO> getDTO() {
+        IssueEntity[] entities = this.activeObjects.find(IssueEntity.class, Query.select());
+        return Arrays.stream(entities).map(mapper::toIssueDTO).collect(Collectors.toList());
+    }
+    public void persistComment(IssueEvent evIssue){
```

Comment:
This method copies logic from `persistIssueEntity`. Do you really need this method